Proofreading-OV-Runtime (#11658)
openvinotoolkit/openvino: docs/OV_Runtime_UG/performance_hints.md

The hints, in contrast, respect the actual model, so the parameters for optimal throughput are calculated for each model individually (based on its compute versus memory bandwidth requirements and capabilities of the device).

## Performance Hints: Latency and Throughput
As discussed in the [Optimization Guide](../optimization_guide/dldt_optimization_guide.md) there are a few different metrics associated with inference speed.
Throughput and latency are some of the most widely used metrics that measure the overall performance of an application.

Therefore, in order to ease the configuration of the device, OpenVINO offers two dedicated hints, namely `ov::hint::PerformanceMode::THROUGHPUT` and `ov::hint::PerformanceMode::LATENCY`.
A special `ov::hint::PerformanceMode::UNDEFINED` hint acts the same as specifying no hint.

For more information on conducting performance measurements with the `benchmark_app`, refer to the last section in this document.

> > **NOTE:** A typical model may take significantly more time to load with `ov::hint::PerformanceMode::THROUGHPUT` and consume much more memory, compared to `ov::hint::PerformanceMode::LATENCY`.

Reviewer: ```suggestion
Keep in mind that a typical model may take significantly more time to load with the `ov::hint::PerformanceMode::THROUGHPUT` and consume much more memory, compared to the `ov::hint::PerformanceMode::LATENCY`.
```

## Performance Hints: How It Works
Internally, every device "translates" the value of the hint to the actual performance settings.

> For example the `ov::hint::PerformanceMode::THROUGHPUT` selects number of CPU or GPU streams.

Reviewer: ```suggestion
For example, the `ov::hint::PerformanceMode::THROUGHPUT` selects number of CPU or GPU streams.
```

Additionally, the optimal batch size is selected for the GPU and the [automatic batching](../OV_Runtime_UG/automatic_batching.md) is applied whenever possible. 

> (To check whether the device supports it, refer to the [devices/features support matrix](./supported_plugins/Device_Plugins.md) article.)

Reviewer: ```suggestion
```
